lib/MetaCPAN/Script/Mapping.pm

```diff
@@ -619,12 +983,35 @@ with B<Exit Code> C< 1 >.
 
 See L<Method C<deploy_mapping()>>
 
-See L<Method C<verify_mapping()>>
+See L<Method C<check_mapping()>>
 
 See L<Method C<MetaCPAN::Role::Script::check_health()>>
 
 =back
 
+=cut
+
+#<<<<<<< HEAD
```

Comment:
Some left over conflict markers.

Reply:
Thank you for your interest in my work.

Yes, you are right. 
I looked into it and fixed the block of documentation which was meant to explain the `--delete --all` option.

```
    Option "--all"
        This option is only effective in combination with Option "--delete".
        It uses the information gathered by
        "MetaCPAN::Role::Script::check_health()" to delete ALL indices in
        the *ElasticSearch* Cluster. This option is usefull to reconstruct a
        broken *ElasticSearch* Cluster

            bin/metacpan mapping --delete --all

        See "Option "--delete""

        See "Method "deploy_mapping()""

        See "Method "MetaCPAN::Role::Script::check_health()""
```

Reply:
As documented in the related issue at
[Fix Corrupted Indices](https://github.com/metacpan/metacpan-api/issues/1048)
this will only work if the **Environment Variables** `PLACK_ENV` or `MOJO_MODE` are set to `development` or `testing`.
Otherwise this will throw an **Exception** `[ 1 ] - EPERM - Operation not permitted`.